src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CountFalsePositives.java

```diff
@@ -32,62 +30,101 @@
 /**
  * Created by tsato on 12/28/16.
  */
-public class CountFalsePositives extends CommandLineProgram {
-    @Argument(doc = "a line-delimited list of mutect2 vcf files", fullName= "", shortName = "V", optional = false)
-    protected File vcfList;
 
-    @Argument(doc = "output file", fullName= "output", shortName = "O", optional = false)
-    protected File output;
+/** TODO: this class should be a variant walker. That way we can ask questions like how are the false positive rates
+ *  over these intervals and so on. For now, we look at the entire vcf and assume ICE intervals
+ */
+
+public class CountFalsePositives extends VariantWalker {
+    @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME, shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME, doc = "Output file (if not provided, defaults to STDOUT)", common = false, optional = true)
+    private File outputFile = null;
+
+    private PrintStream outputStream = null;
+
+    long targetTerriroty = 0;
+    long indelFalsePositiveCount = 0;
```

Comment:
I'm of the school of thought that you should try to make as many variables `final` as possible -- this indicates that either it's never changed at all after assignment or that it is mutated but only according to its member methods.  Therefore, I would make these `MutableLong`s and replace `++` with `increment()` below.  The downside is that using an object instead of a primitive is kind of prissy.  I'll let you choose.